policies/election-procedures.md

```diff
@@ -0,0 +1,164 @@
+# Astropy Election Procedures
+
+## Overview
+
+Astropy votes are managed with the [Helios Voting platform](https://heliosvoting.org) and
+announced publicly via the astropy-dev mailing list and the OpenAstronomy Discourse forum. 
+
+## Procedure for creating an election
+
+1.  Set a date for the election and announce the election
```

Comment:
Might want to change each number into `###` sub-section as GitHub isn't rendering the `ul` listing among the numbering nicely.

Reply:
Fixed the bullets.